lottie-compose/src/main/java/com/airbnb/lottie/compose/LottieAnimation.kt

```diff
@@ -131,7 +133,11 @@ fun LottieAnimation(
             drawable.maintainOriginalImageBounds = maintainOriginalImageBounds
             drawable.clipToCompositionBounds = clipToCompositionBounds
             drawable.clipTextToBoundingBox = clipTextToBoundingBox
-            drawable.progress = progress()
+            if (!drawable.animationsEnabled(context) && drawable.markerForAnimationsDisabled != null) {
+                drawable.progress = drawable.markerForAnimationsDisabled!!.startFrame
```

Comment:
```suggestion
            val markerForAnimationsDisabled = drawable.markerForAnimationsDisabled
            if (!drawable.animationsEnabled(context) && markerForAnimationsDisabled != null) {
                drawable.progress = markerForAnimationsDisabled.startFrame
```

Let's avoid `!!` here with a smart cast